setup.py

```diff
@@ -107,8 +107,11 @@ def build_extensions(self):
             compiler_make = 'msvc'
 
         if compiler_type != 'msvc':
-            new_flags = {"-Wextra": None, "-std": "c11",
-                         "-march": "native", "-mtune": "native"}
+            # Prevent generation of code for the compiling machine (for the moment)
+            # This is harmful when building a binary package.
+            # new_flags = {"-Wextra": None, "-std": "c11",
+            #       "-march": "native", "-mtune": "native"}
+            new_flags = {"-Wextra": None, "-std": "c11"}
```

Comment:
Can you add a new option for adding this back (see `user_options` above)?

Reply:
I have no idea how to include a new option systematically.
Should we introduce a command-line option to `setup.py`?
Or, how about checking an environment variable `XPREC_NATIVE_OPT`?